libraries/apollo-api/api/apollo-api.api

```diff
@@ -713,6 +713,12 @@ public final class com/apollographql/apollo/api/ImmutableMapBuilder {
 	public final fun put (Ljava/lang/Object;Ljava/lang/Object;)Lcom/apollographql/apollo/api/ImmutableMapBuilder;
 }
 
+public final class com/apollographql/apollo/api/IncrementalResultIdentifierKt {
+	public static final fun isPending (Ljava/util/Set;)Z
+	public static final fun nonPending (Ljava/util/Set;)Ljava/util/Set;
+	public static final fun pending (Ljava/util/Set;)Ljava/util/Set;
+}
```

Comment:
Can those be removed from public API? `isPending` can be internal I think, `nonPending` can be moved to `apollo-runtime` and `pending` is only used in tests.